src/Test/QuickCheck/StateMachineModel/Example.hs

```diff
@@ -217,7 +217,7 @@ prop_parallel prb = parallelProperty
 
 scopeCheck
```

Comment:
Similarly this should also move out together with `debugMem` I'm not sure what to do about the properties that follow though. It would be nice to make sure that client code doesn't even know about `Pid` (or `Ref` for that matter, or even further it would be nice if the client doesn't even know that a program is `[Untyped cmd XXX]` 

So I don't think we can completely hide `Untyped` from the client, but it should probably only be used for `gens` and possibly `shrink1`